app/views/course/dashboards/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1 class="mb-3">Hi, <%= current_user.email %></h1>
+<%= link_to "Learn", course_section_path(@section), class: "btn btn-primary" %>
```

Comment:
Cool. I like how you incorporated the link to the section. We're gonna need to change this to be a more dynamic on another PR, but that's a good start.

Reply:
Definitely. I included it as a sort of placeholder for now!